content/en/network_monitoring/devices/setup.md

```diff
@@ -43,14 +43,14 @@ To monitor individual devices:
 
     ```yaml
     init_config:
-      loader: core                     # use core check implementation of SNMP integration. recommended
```

Comment:
note for reviewers: removing some spaces to avoid comment spanning visually on multiple lines:

![Uploading image.png…]()